staging/src/k8s.io/kubectl/pkg/generate/versioned/ingress.go

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
Not needed, drop it.

Reply:
I don't understand 😕Dropping the first line would break the comment?

Reply:
This file is not needed, since you have dedicated method in create ingress.